content/docs/join_us/recruitment_FAQs.md

```diff
@@ -19,29 +19,49 @@ Questions are grouped by:
 
 - [Application Stage](#application-stage)
 - [Profile](#profile)
-- [Position](#position)
+- [Role](#role)
 - [Interview](#interview)
 - [Offer](#offer)
 
 ## Application Stage
 
-### Is the position open to applicants outside of the UK who do not currently have a UK working Visa but could obtain one?
+### Is the position open to applicants outside of the UK who do not currently have a UK work visa?
 
-Yes, the positions are open to applicants outside the UK and our HR team ([HR@turing.ac.uk](mailto:HR@turing.ac.uk)) will be able to offer you additional information on the process of obtaining a working Visa.
+Yes, the positions are open to applicants outside the UK.
+If you do not currently have the Right to Work in the UK, the Turing will sponsor a Skilled Worker visa application (see [below](#does-the-turing-cover-relocation-and-visa-costs) for details).
+If you have general questions about the process, feel free to reach out to the [REG recruitment team][recruitment]; however, once your application has started, you should direct any questions about your specific situation to the Resourcing Officer who is handling your application.
 
 ### Are you accepting remote applicants?
 
 Members need to be based in the UK.
 However, you do not need to be in the London office every day: almost all of our work is done in a hybrid manner, and we have team members as far away as Scotland and the South West of England.
```

Comment:
I would write this section as the follows:

> Members need to be based in the UK. However, you do not need to be in the London office every day nor on a regular weekly schedule, as almost all of our work is done in a hybrid manner. We have team members that live further away in places such as Scotland and the South West of England that come to the office less frequently. The on paper expectation is that it would not be unreasonable to ask team members to spend up to four days a month in the office, but in practice we have been able to support a range of hybrid working patterns with fewer days in the office. We prefer to let the work to be done dictate the most effective collaboration style for carrying it out, and thus there may be some reasonable variation across projects in terms of requirements for in-office work. We also hold two in-person team events per year, the Team Away Day in the winter and Hack Week in June.

I note that we don't have a page describing either the Away Day or the Hack Week in the handbook (yet), but issue #20 talks about this so I've dropped a comment there to link to the FAQ page.